src/main/cpp/RLBotInterface/RLBotMessages/MutexUtilities/MutexUtilities.cpp

```diff
@@ -0,0 +1,94 @@
+#include "MutexUtilities.hpp"
+
+#include <Windows.h>
```

Comment:
One day when we support Linux I'll replace this with a boost mutex >:)

This looks good though!